shared/lib-metrics/src/main/java/gov/va/vro/metricslogging/MetricLoggerService.java

```diff
@@ -43,6 +59,15 @@ public ArrayList<String> getTagsForSubmission(String[] customTags) {
     if (customTags != null) {
       tags.addAll(Arrays.asList(customTags));
     }
+
+    tags.add("env:" + env);
+    tags.add("itportfolio:" + itPortfolio);
+    tags.add("team:" + team);
+    tags.add("service:" + service);
+    if (dependency != null && !dependency.isEmpty()) {
```

Comment:
since we use apache commons this line:
`if (dependecy != null && !dependency.isEmpty()) {`
can be replaced with
`if (StringUtils.isNotBlank()) {`